google-cloud-dlp/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Release History
 
+### 0.9.0 / 2019-04-09
+
+* Extract gRPC header values from request
```

Comment:
I would move this to right before the documentation lines at the end. This is not the most important change of the release.